src/app/shared/index.ts

```diff
@@ -1,9 +1,15 @@
+export { DateInputComponent } from './components/date-input/date-input.component';
 export { DurationInputComponent } from './components/duration-input/duration-input.component';
 export { MessageComponent } from './components/message/message.component';
+export { MultiAutocompleteInputComponent } from './components/multi-autocomplete-input/multi-autocomplete-input.component';
 export { SearchInputComponent } from './components/search-input/search-input.component';
 
 export { MockDurationInputComponent } from './components/duration-input/duration-input.component.mock';
 export { MockSearchInputComponent } from './components/search-input/search-input.component.mock';
+export { MockMultiAutocompleteInputComponent } from './components/multi-autocomplete-input/multi-autocomplete-input.component.mock';
+export { MockDateInputComponent } from './components/date-input/date-input.component.mock';
 
 export { OrderByPipe } from './pipes/order-by/order-by.pipe';
 export { TimeDurationPipe } from './pipes/time-duration/time-duration.pipe';
+
+export { MultiAutocompleteInputValidator } from './validators/multi-autocomplete-input.validator';
```

Comment:
you can use `export * from 'path'` - this will allow you to write one export line per file.